### PR TITLE
 update auth section

### PR DESCRIPTION
**Pull request by s00uvik**

### CODE REVIEW

**Comment by s00uvik** [comment]
This PR have previous PR's 3 files, 
forgot to create auth-update branch from dev branch.

File: src/app/(user-dashboard)/dashboard/(compant-settings)/company-tasks/page.tsx [hunk 1]
```diff
@@ -165,29 +165,27 @@ const TaskPage = ({ params }: { params: { id: string } }) => {
...
-                  approveTaskCompletion={approveTaskCompletion}
-                  isLoading={isLoadingApprovedTask}
-                />
-              ))
-            )}
-          </div>
-        )
+      {isLoadingTask && <TaskSkeleton />}
+
+      {taskFilter === "assign-task" || (
```
Comment (Piyush-Kumar-1) [inline src/app/(user-dashboard)/dashboard/(compant-settings)/company-tasks/page.tsx:170]: Add error condition

File: src/app/auth/sign-in/page.tsx [hunk 2]
```diff
@@ -45,107 +52,49 @@ const SignIn: React.FC = () => {
...
-  const handleLogoClick = () => {
-    router.push("/");
-  };
-
   return (
-    <div className="flex min-h-screen  justify-center bg-white -m-4">
+    <section className="flex min-h-screen  justify-center bg-white -m-4">
       <div className="w-full m-4 max-w-sm md:mt-10 px-4 sm:px-0">
-        <div className="flex justify-center m-2 cursor-pointer" onClick={handleLogoClick}>
+        <div className="flex justify-center m-2 cursor-pointer" onClick={() => router.push("/")}>
```
Comment (Piyush-Kumar-1) [inline src/app/auth/sign-in/page.tsx:72]: This should be button instead of div

File: src/app/auth/sign-in/page.tsx [hunk 3]
```diff
@@ -45,107 +52,49 @@ const SignIn: React.FC = () => {
...
-              } focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500`}
-              disabled={isSubmitting}
-            >
-              {isSubmitting ? "Signing In..." : "Sign In"}
-            </Button>
-          </form>
           <div className="mt-4 mb-4 text-center text-sm md:text-normal">
             <span className=" text-slate-600">
               Don&apos;t have an account yet?{" "}
               <span
```
Comment (Piyush-Kumar-1) [inline src/app/auth/sign-in/page.tsx:87]: button instead of span

File: src/lib/api/auth/auth.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  Cookies.remove("accessToken");
+  Cookies.remove("refreshToken");
+  localStorage.clear();
+  Cookies.remove("isClient");
+  location.reload();
+
+  return null;
+};
+
+export const verifyToken = async (token: RequestCookie | undefined) => {
```
Comment (Piyush-Kumar-1) [inline src/lib/api/auth/auth.ts:56]: Where is this being used?

**Comment by Piyush-Kumar-1** [comment]
Start writing comments. It is difficult for others to understand code without comment. 
Take a look at this https://www.freecodecamp.org/news/best-practices-for-react/

**Pull request merged**